libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -0,0 +1,80 @@
+import { getConfig, localizeLink } from '../../../utils/utils.js';
+
+export function toFragment(htmlStrings, ...values) {
+  const templateStr = htmlStrings.reduce((acc, htmlString, index) => {
+    if (values[index] instanceof HTMLElement) {
+      return `${acc + htmlString}<elem ref="${index}"></elem>`;
+    }
+    return acc + htmlString + (values[index] || '');
+  }, '');
+
+  const fragment = document.createRange().createContextualFragment(templateStr).children[0];
+
+  Array.prototype.map.call(fragment.querySelectorAll('elem'), (replaceable) => {
+    const ref = replaceable.getAttribute('ref');
+    replaceable.replaceWith(values[ref]);
+  });
+
+  return fragment;
+}
+
+export function debounceCallback(callback, time = 200) {
+  if (typeof callback !== 'function') return undefined;
+
+  let timeout = null;
+
+  return (...args) => {
+    clearTimeout(timeout);
+    timeout = setTimeout(() => callback(...args), time);
+  };
+}
+
+// TODO: should we replace these with proper Locale/country service(s)?
+export function getLocale() {
+  return document.documentElement.getAttribute('lang') || 'en-US';
+}
+
+export function getCountry() {
```

Comment:
not really sure we need this method either since it's only being used in one place (getting the helpx link)

Reply:
Moved to search for now